### PR TITLE
Add a note about custom app setting for newly created tenants

### PR DESCRIPTION
**Pull request by therealjohn**
Newly created tenants from the dev program will have this setting on by default. Added an important alert for this.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md](https://github.com/MicrosoftDocs/msteams-docs/blob/49a5d891ef325942b7bb583c12dad7648540be6b/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fprepare-your-o365-tenant.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0c56b84b-6ab6-4d2d-9ad5-4a63f9771b62) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/zone-pivot-groups.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/49a5d891ef325942b7bb583c12dad7648540be6b/msteams-platform%2Fzone-pivot-groups.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/02c97bbe-c3db-46cc-b811-2d4d644dcbfe) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ Enable Teams for your organization and for more information, see [enabling Teams
 
 ## Enable custom Teams apps and turn on custom app uploading
 
+> [!IMPORTANT]
+> You can skip the steps below if you chose the Instant Sandbox option with the Teams sample data pack which will automatically enable the **Upload custom apps** setting.
```
Comment (Lauragra) [inline msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md:34]: Suggested revision: "You do not have to follow the steps to enable sideloading if you have an instant sandbox or a configurable sandbox with the Teams sample data pack installed. The **Upload custom apps** setting is enabled by default in the Teams sample data pack."

Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md]: @Lauragra 

We have updated the note in the article as per your suggestion. Request you to review the changes and approve.

image

File: msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ Enable Teams for your organization and for more information, see [enabling Teams
 
 ## Enable custom Teams apps and turn on custom app uploading
 
+> [!IMPORTANT]
+> If you have installed an instant sandbox or a configurable sandbox with the Teams sample data pack, you can skip the steps to enable sideloading. The **Upload custom apps** setting is enabled by default in the Teams sample data pack.
```
Comment (Lauragra) [inline msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md:34]: I suggest changing "If you have installed an instant sandbox" to "If you set up a Microsoft 365 Developer Program instant sandbox".

Comment (v-ypalikila) [reply msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md]: updated as suggested.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a8b3db5fe227101782e7d3a1588e606b6cc4eee/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fprepare-your-o365-tenant.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37d9da24-c3bf-43c5-b576-d5298f3fa160) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/prepare-your-o365-tenant.md](https://github.com/MicrosoftDocs/msteams-docs/blob/047bc4f3eb904619640506c78e7aa60ee64b0d44/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fprepare-your-o365-tenant.md) | 92 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c06826a-8dfc-4432-906c-ac32572fdee3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Pull request merged**